name.abuchen.portfolio/src/name/abuchen/portfolio/online/impl/FinnhubSearchProvider.java

```diff
@@ -0,0 +1,78 @@
+package name.abuchen.portfolio.online.impl;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import name.abuchen.portfolio.Messages;
+import name.abuchen.portfolio.online.SecuritySearchProvider;
+
+/**
+ * Use the <a href="https://finnhub.io/">Finnhub</a> API to search for securities.
+ * 
+ * @see FinnhubSymbolSearch
+ */
+public class FinnhubSearchProvider implements SecuritySearchProvider
```

Comment:
Why do you split the code into two classes: FinnhubSearchProvider and FinnhubSymbolSearch? The former is just calling the latter. I think it will be easier to read (and maintain in the future) if it is just one class.

Reply:
I will combine them. Again, I copied the Yahoo code and did not refactor it.